### PR TITLE
Fix ssl certificate error and broken Hive download link

### PR DESCRIPTION
**Pull request by ebyhr**
```
Could not fetch URL https://pypi.python.org/simple/supervisor/: There was a problem confirming the ssl certificate: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:618) - skipping
Could not find a version that satisfies the requirement supervisor (from versions: )
```

### CODE REVIEW

**Comment by ebyhr** [comment]
Hit another issue:
```
 > [ 4/10] RUN curl -fLsS -o /tmp/hive.tar.gz --url https://downloads.apache.org/hive/hive-3.1.2/apache-hive-3.1.2-bin.tar.gz &&     tar xzf /tmp/hive.tar.gz --directory /opt && mv /opt/apache-hive-3.1.2-bin /opt/hive:
0.890 curl: (22) The requested URL returned error: 404 Not Found
```

File: testing/hive3.1-hive/Dockerfile [hunk 1]
```diff
@@ -36,11 +36,11 @@ RUN yum install -y \
 ENV JAVA_HOME=/usr/lib/jvm/zulu-8
 
 ARG HADOOP_VERSION=3.1.2
-ARG HIVE_VERSION=3.1.2
+ARG HIVE_VERSION=3.1.3
```
Comment (anusudarsan) [inline testing/hive3.1-hive/Dockerfile:39]: should we run some tests locally with this before merging?

**Comment by anusudarsan** [comment]
the ssl related fix LGTM

File: testing/hive3.1-hive/Dockerfile (hunk 1 above)
Comment (ebyhr) [reply testing/hive3.1-hive/Dockerfile:39]: Yes, I ran the following command and all tests passed:
```
testing/bin/ptl suite run --config config-apache-hive3 --suite suite-hms-only
... 
tests               | ===============================================
tests               | tempto-tests
tests               | Total tests run: 81, Failures: 0, Skips: 0
tests               | ===============================================
```

**Review by wendigo — APPROVED** [review]

**Pull request merged**